lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -568,6 +566,12 @@ func convertContainerToContainerInfo(c *exec.Container) *models.ContainerInfo {
 	info.ContainerConfig.ImageID = container.ExecConfig.ImageID
 	info.ContainerConfig.RepoName = &container.ExecConfig.RepoName
 	info.ContainerConfig.CreateTime = container.ExecConfig.CreateTime
+
+	// Convert create timestamp to nanoseconds for older containers that use seconds
+	if info.DataVersion < feature.ContainerCreateTimestampVersion {
```

Comment:
This conversion should be in `lib/migration/plugins` and following the approach documented in `lib/migration/plugins/plugins.md`

While this is a very simple conversion we shouldn't have migration logic scattered and inline in the rest of the code.

Reply:
Fixed - thanks!